### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#602] [Galaxy Trivia] [answered]
**DEBAI**
**ONMIRAMIGIT**
**SARNFRET**
**VEERMO**

**Comment by peterbaccaro** [comment]
ABIDE
IMMIGRATION
TRANSFER
REMOVE

The odd one out is **IMMIGRATION** as it is a noun whereas "ABIDE", "TRANSFER", and "REMOVE" are all verbs.

**Comment by Veercodeprog** [comment]
DEBAI as it forms Abide which is unrelated to other words,that are related to travelling
 IMMIGRATION
TRANSFER
REMOVE

**Comment by thuongtruong1009** [comment, marked as answer]
The odd one out is "VEERMO" because the letters in the other three options can be rearranged to form names of countries:

DEBAI -> "BAHDEI" -> "BAHRAIN"
ONMIRAMIGIT -> "MONGRAMITI" -> "MONGOLIA"
SARNFRET -> "TRANSFER" -> "FRANCE"
However, the letters in "VEERMO" cannot be rearranged to form the name of a country.

**Comment by ParanoidUser** [comment]
**Hint:** What role can each word play in a sentence?
